docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -58,21 +65,49 @@ This guide introduces you to the basics of human task orchestration. You will cr
 </TabItem>
 </Tabs>
 
-Take the following five steps to create and run your first process with a human in the loop:
+<Tabs groupId="install" className="tabs-hidden">
+<TabItem value="saas">
+
+You will learn how to:
+
+- Create and run your first process with a human in the loop.
+- Build a BPMN diagram using Web Modeler.
+- Create and link a form to a user task.
+- Validate your process using Play mode.
+
+<p><img src={HumanTaskDiagramImg} alt="The completed BPMN diagram" /></p>
+
+</TabItem>
+<TabItem value="sm">
+
+You will learn how to:
+
+- Create and run your first process with a human in the loop.
+- Build a BPMN diagram using Modeler.
+- Create and link a form to a user task.
+- Deploy and run your process.
+- Complete the task in Tasklist, and check the process in Operate.
+
+</TabItem>
+</Tabs>
+
+Perform the following steps to create and run your first process with a human in the loop.
 
 ## Step 1: Create a new process
 
-In this step, you will design a process that demonstrates how to route the process flow based on a user decision. In this example, you will create a process to decide what is for dinner.
+Start by designing a process to demonstrate how to route the process flow based on a user decision. In this example, you will create a process to decide what to eat for dinner.
 
 ### Create a new file
 
 <Tabs groupId="install" className="tabs-hidden">
 <TabItem value="saas">
 
-1. Every file in Web Modeler requires a project. Within Modeler, click **New project**.
-2. Name your project and select **Create new > BPMN diagram**.
-3. Give your file a descriptive name. In this case, name it `Decide for Dinner`.
-4. Make sure to name the process itself as well. Click the empty canvas, and specify the process name and technical ID in the properties panel. This specifies how the process will appear in other tools of Camunda 8.
+First, you must create the BPMN diagram file and project.
+
+1. Open Web Modeler, and click **New project**. Every file in Web Modeler requires a project.
+2. Select **Create new > BPMN diagram**.
```

Comment:
For automation, users should always use process applications. They have the most features, including a smoother Play deployment experience